src/elisp/treemacs-icons.el

```diff
@@ -134,6 +138,9 @@ Remember the value in `treemacs--default-icons-alist'."
 (defvar treemacs-icon-tag-leaf-text        (eval-when-compile (propertize "• " 'face 'font-lock-constant-face)))
 (defvar treemacs-icon-tag-node-closed-text (eval-when-compile (propertize "▸ " 'face 'font-lock-string-face)))
 (defvar treemacs-icon-tag-node-open-text   (eval-when-compile (propertize "▾ " 'face 'font-lock-string-face)))
+(defvar treemacs-icon-error-text           (eval-when-compile (propertize "• " 'face 'font-lock-string-face)))
```

Comment:
I wonder if this eval-when-compile even does anything. I gone a bit overboard with the whole compile-time-evaluation thing when I tried making loading of icons faster.